RELEASE_NOTES.md

```diff
@@ -1,5 +1,31 @@
 # ChefDK 3.7 Release Notes
 
+## Chef 14.9.13
+
+ChefDK now ships with Chef 14.9.13. See <https://docs.chef.io/release_notes.html> for more information on what's new.
+
+## InSpec 3.3.14
+
+ChefDK now ships with Inspec 3.3.14. See <https://github.com/inspec/inspec/releases/tag/v3.3.14> for more information on what's new.
+
+## Updated Components and Tools
+
+### kitchen-inspec 1.0.1
+
+Support bastion configuration in transport options
+
+### Other updates
+
+* `bundler`: 1.16.1 -> 1.17.3
+* `chef-apply`: 0.2.4 -> 0.2.7
+* `chefstyle`: 0.11.0 -> 0.12.0
```

Comment:
We don't want people using this so we shouldn't tell them about it.

Reply:
Will remove.